src/wallet/wallet.cpp

```diff
@@ -2590,7 +2590,9 @@ std::map<CTxDestination, std::vector<COutput>> CWallet::ListCoins(interfaces::Ch
         if (it != mapWallet.end()) {
             int depth = it->second.GetDepthInMainChain(locked_chain);
             if (depth >= 0 && output.n < it->second.tx->vout.size() &&
-                IsMine(it->second.tx->vout[output.n]) == ISMINE_SPENDABLE) {
+                IsMine(it->second.tx->vout[output.n]) ==
```

Comment:
a16d2eb0d593633d6ba733d761c56c835b0e30dc

Does it make sense to do something like:
```cpp
    // some comment here
    isminetype is_mine_filter = IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)
                              ? ISMINE_WATCH_ONLY
                              : ISMINE_SPENDABLE;

…
                IsMine(it->second.tx->vout[output.n]) == is_mine_filter) {
```

Reply:
Done